src/negocio/ControladorEmpregado.java

```diff
@@ -19,10 +22,48 @@ public void cadastrarEmpregado() {
 
     }
 
-    public boolean autenticarEmpregado(Empregado empregado) {
-        boolean valido = false;
+    public boolean autenticarEmpregado(String _email, String _senha) {
+        boolean emailValidado = false;
+        boolean validado = false;
+        Empregado eSenha = null;
+        String stringSenha = " ";
 
-        return valido;
+        for(Empregado e : this.repoEmpregado.listar()){
+            if(e.getEmail().equals(_email)){
+                eSenha = e;
```

Comment:
Aqui você pode fazer o seguinte para evitar usar o objeto Empregado: Cria um novo objeto `String stringSenhaCadastro`.

```suggestion
                stringSenhaCadastro = e.getSenha();      
```